### ISSUE TITLE
Decreate feature gap between sccache and cachepot

### ISSUE THREAD

**Issue by Xanewok** [paritytech/cachepot#155]
Since we forked, there has been some improvements made here but also we missed out on some of the PRs made against sccache (after the project got more action since the fork). This will be beneficial by getting new features/bug fixes but also will facilitate upstreaming some of our improvements as well.

**Comment by TriplEight** [comment]
As proposed at the meeting:

# MVP for sccache in CI/CD

- [ ] finer cache hits tracking and non-hits
- [ ] overlay and docker support
- [ ] rust bugs worked around: mtime and fullpath cache invalidation issues

# Outside of MVP

- distributed compilation for CI
- encrypted/signed shared artifacts
- undep OpenSSL
- metrics support

# Stuff not related to the CI use-case, but nonetheless important

- distributed compilation for devs

cc @drahnr @kirushik

**Comment by equal-l2** [comment]
Hello, any progress in here?

**Comment by drahnr** [comment]
@Xanewok will be chipping away on that in the long run, I'll be focused on merging and reviewing upstream PRs. So there is no timeframe right now when which fix will be upstreamed.

**Comment by abhiss** [comment]
> We will be winding down cachepot as a separate project and instead plan to upstream all of our fixes to the sccache project, now that we have maintainer access and help maintain the crate and publish it. This issue tracks the required work.

Does this mean cachepot is basically deprecated and we should use sccache instead? I just want to confirm since it looks like this project is still fixing bugs and updating dependencies.

**Comment by Xanewok** [comment]
Yes, since sccache increased its dev bandwidth and we can actually upstream the work, I think it's fair to say you should use sccache now, as we merge our stuff there.
